include/linux/simd_x86.h

```diff
@@ -442,6 +450,35 @@ zfs_bmi2_available(void)
 #endif
 }
 
+/*
+ * Check if AES instruction set is available
+ */
+static inline boolean_t
+zfs_aes_available(void)
```

Comment:
nit: can we rename this to `zfs_aes_instruction_available()` or something? This name makes it sound like encryption is completely impossible on this system.

Reply:
For consistentcy with the other instructions checks I'd prefer we leave the names as is.